daemon/cluster/services.go

```diff
@@ -262,6 +262,11 @@ func (c *Cluster) ServiceLogs(ctx context.Context, input string, config *backend
 		c.mu.RUnlock()
 		return err
 	}
+	// TODO(dperny) can I trust that service.Spec.Task always exists?
+	container := service.Spec.Task.GetContainer()
+	if container.TTY {
+		return errors.New("Service logs not supported on tasks with TTY attached")
```

Comment:
`Service` should be lowercase.

Maybe `with a TTY attached` sounds slightly better?